code/_onclick/hud/screen_objects.dm

```diff
@@ -265,9 +265,12 @@
 	screen_loc = "7,7 to 10,8"
 
 
-/obj/screen/storage/New(new_master, mapload)
+/obj/screen/storage/Initialize(mapload)
 	. = ..()
-	master = new_master
+	if(!isitem(loc))
```

Comment:
This is likely incompatible with storage component

Reply:
We don't have one yet, and not sure if we'll implement it. /tg/ storage is quite different from our bay-based variant and not compatible.